docs/en/latest/concepts/annotations.md

```diff
@@ -204,3 +204,54 @@ spec:
             port:
               number: 80
 ```
+
+Use ApisixPluginConfig
+---------
+
+You can use the following annotations to use the `ApisixPluginConfig`.
+
+* `k8s.apisix.apache.org/plugin-conifg-name`
+  
+If this annotations set to `ApisixPluginConfig.metadata.name` the route will enable websoket
+
+For example, the following Ingress, if we set `k8s.apisix.apache.org/plugin-conifg-name: "echo-and-cors-apc"`. `/api/*` route will enable echo-and-cors-apc
```

Comment:
```suggestion
As an example, we attach the annotation `k8s.apisix.apache.org/plugin-conifg-name: "echo-and-cors-apc` for the following Ingress resource, so that `/api/*` route will enable the [echo](https://apisix.apache.org/docs/apisix/plugins/echo/) and [cors](https://apisix.apache.org/docs/apisix/plugins/cors/) plugins.
```